src/screens/FeedbackScreen.js

```diff
@@ -149,6 +164,9 @@ export const FeedbackScreen = () => {
 };
 
 const styles = StyleSheet.create({
+  inputBorderContrastColor: {
```

Comment:
please do not name it `...Color`, as it is a style and can change its content. if that happens, you would not like to change the name. please be accurate with `backgroundContrast` [here](https://github.com/smart-village-solutions/smart-village-app-app/pull/779) and name it `inputContrast` for example

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/449a788e159ff48ac0d38841c88ee5ca5f4baef5